packages/server/graphql/public/typeDefs/ToggleFeatureFlagSuccess.graphql

```diff
@@ -0,0 +1,6 @@
+type ToggleFeatureFlagSuccess {
+  """
+  The feature flag that was toggled
+  """
+  featureFlag: OwnedFeatureFlag!
```

Comment:
curious, why create a new type vs. having `{enabled, featureFlag}`?

Reply:
I think we need it [here](https://github.com/ParabolInc/parabol/blob/93fa3cf3231c560b7b455ce4b6d68408711cc11a/packages/client/modules/userDashboard/components/OrgBilling/OrgFeatureFlags.tsx). 

In `ToggleFeatureFlagSuccess`, we could do what you're suggesting, but in `Organization`, I think we do have to return a new type? I went with your suggestion [here](https://github.com/ParabolInc/parabol/pull/10446#discussion_r1842984643).

Reply:
ah! that makes sense. LGTM